master/internal/core.go

```diff
@@ -740,6 +742,8 @@ func (m *Master) Run(ctx context.Context) error {
 	// Initialize the HTTP server and listen for incoming requests.
 	m.echo = echo.New()
 	m.echo.Use(middleware.Recover())
+	tp := otelConfig()
+	m.echo.Use(otelecho.Middleware("determined-master", otelecho.WithTracerProvider(tp)))
```

Comment:
Did we test whether it's necessary to pass the tracer provider here? I think this might be belt-and-braces, so it's not necessarily an issue, just curious.

Reply:
I will try that!